submissions/ArthurGorbenko/html.css.practice/index.html

```diff
@@ -11,168 +11,165 @@
+                           </a>
+                       </li> 
+                       <li class="card">
+                          <a href="" class="link">
+                              <img src="news.png" class="icon">
+                           <span class="title_btn">News</span>
+                           </a>
+                       </li> 
+                  </ul>
+
+                <div class="even_more"><a href="" class="link">Even more from G</a></div>
+            </div>
+        <label for="more" id="more_label" class="button">
+            <div class="more_button">More</div>
+        </label>
+      <ul class="list_1">
+       <li class="card">
+           <a href="" class="link">
+            <img src="earth.png" class="icon">
+                <span class="title_btn">Earth</span>
+            </a>
+        </li>
+       <li class="card">
+           <a href="" class="link">
+           <img src="drive.png" class="icon">
+                <span class="title_btn">Disk</span>
+            </a>
+        </li>
+       <li class="card">
+           <a href="" class="link">
+               <img src="app_market.png" class="icon">
+                    <span class="title_btn">App Market</span>
+            </a>
+        </li>
+       <li class="card">
+           <a href="" class="link">
+               <img src="blog.png" class="icon">
+            <span class="title_btn">My Blog</span> 
+          </a>
+        </li> 
+       <li class="card">
+           <a href="" class="link">
+               <img src="bookmarks.png" class="icon">
+            <span class="title_btn">Bookmarks</span> 
+           </a>
+        </li> 
+       <li class="card">
+           <a href="" class="link">
+               <img src="calendar.png" class="icon">
+            <span class="title_btn">Calendar</span> 
+           </a>
+        </li> 
+       <li class="card">
+           <a href="" class="link">
+            <img src="cloud_search.png" class="icon">
+                <span class="title_btn">Cloud</span>
+            </a>
+        </li> 
+       <li class="card">
+           <a href="" class="link">
+             <img src="documents.png" class="icon">
+                <span class="title_btn">Documents</span>
+             </a>
+        </li> 
+       <li class="card">
+           <a href="" class="link">
+               <img src="maps.png" class="icon">
+                    <span class="title_btn">Maps</span>
+            </a>
+        </li> 
+       <li class="card">
+           <a href="" class="link">
+               <img src="gmail.png" class="icon">
+            <span class="title_btn">Mail</span>
+            </a>
+        </li> 
+    </ul>
+   </div>
+        <div class="ring"><a href="" class="link"><img src="notification.png" class="icon" id="ring_img"></a></div>
+        <div class="Profile"><a href="" class="link"><img src="profile_picture.png" class="icon" id="profile_img"></a></div>
+    </main>
 </body>
 </html>
```

Comment:
Add an empty line at the end of every file.

Reply:
[Added.](https://github.com/kottans/frontend-2019-p2p/pull/67/commits/0a5c6102a4743aae8508d48acdf15ae8bb025615)

Reply:
[Added](https://github.com/kottans/frontend-2019-p2p/pull/67/commits/0a5c6102a4743aae8508d48acdf15ae8bb025615)